javaparser-core/src/main/java/com/github/javaparser/printer/PrettyPrinterConfiguration.java

```diff
@@ -24,6 +24,7 @@
 public class PrettyPrinterConfiguration {
     private boolean printComments = true;
     private String indent = "    ";
+    private Class<? extends PrettyPrintVisitor> visitorClass;
```

Comment:
After some thinking: maybe an overridable factory method on the configuration would be best, like "createVisitor(config)"